### DISCUSSION TITLE
`pcollections.org` advertises sports betting

### DISCUSSION THREAD

**Discussion by ghost** [hrldcpr/pcollections#117] [General]
Hello,
it looks like the page `https://pcollections.org/`, which was originally the home page of this project, now contains a footer at the bottom left advertising sports betting. In case you still have control over the page, could you maybe shut it down or recreate the redirect (which was removed in 373b72186a8f770b79e1d837792c6c745123ac09)?

Screenshot of the page (not sure how the page looks without ad-blocker):
![Screenshot of footer](https://github.com/user-attachments/assets/3b692a3d-f7c1-46bb-8a4e-a9696ac2df6a)

**Comment by hrldcpr** [comment]
Sigh, thanks for pointing this out; sadly I no longer own the domain so there's not much I can do about it. I've just emailed the registrar in case they're able to help, but it might not actually be against any of their policies.

Thanks for the vigilance!
